### PR TITLE
Update server and scripts

### PR DESCRIPTION
**Pull request by predragnikolic**
* improve the download script. (it will do everything that is required, before it required a bit of manual work, see the commit description for more details)
* update the server to the lastest release
* remove userLanguages from initializationOptions as it is no longer required.

Todo, check if I need to update/add/remove some settings.

### CODE REVIEW

**Comment by predragnikolic** [comment]
closes #8

**Comment by rchl** [comment]
Can we do this for the download script instead?

```sh
#!/usr/bin/env bash

GITHUB_REPO_NAME="tailwindlabs/tailwindcss-intellisense"

# download the release
# more info here - https://gist.github.com/steinwaywhw/a4cd19cda655b8249d908261a62687f8

curl -s "https://api.github.com/repos/${GITHUB_REPO_NAME}/releases/latest" \
| grep "browser_download_url.*vsix" \
| cut -d : -f 2,3 \
| tr -d \" \
| wget -qi -

# get the zip(.vsix) file
tarball="$(find . -name "*.vsix")"

# clean up
rm -rf ./language-server-temp
rm -rf ./language-server
mkdir ./language-server

# unzip
unzip -a "$tarball" -d ./language-server-temp

# ./language-server/package.json is required for lsp_utils to work. Reuse package.json from the extension folder.
cp ./language-server-temp/extension/package.json ./language-server
cp -R ./language-server-temp/extension/dist/server ./language-server

# ./language-server/package-lock.json is required. Without it an error will appear when when starting the plugin.
cd language-server && npm i --production && cd ..

# clean up
rm "$tarball"
rm -rf ./language-server-temp
echo "Done"
```

That's because there seems to be a couple of unnecessary files in the repo left out from the vscode extension.

File: sublime-package.json [hunk 1]
```diff
@@ -110,6 +110,14 @@
                       "type": "number",
                       "default": 16,
                       "markdownDescription": "Root font size in pixels. Used to convert `rem` CSS values to their `px` equivalents. See `#tailwindCSS.showPixelEquivalents#`."
+                    },
+                    "tailwindCSS.inspectPort": {
```
Comment (rchl) [inline sublime-package.json:114]: This is not handled by the server but by the extension.

File: plugin.py [hunk 2]
```diff
@@ -18,7 +18,7 @@ class LspTailwindcssPlugin(NpmClientHandler):
     package_name = __package__
     server_directory = 'language-server'
     server_binary_path = os.path.join(
-        server_directory, 'extension', 'dist', 'server', 'index.js'
+        server_directory, 'server', 'tailwindServer.js'
     )
```
Comment (rchl) [inline plugin.py:22]: Can be one line now.

**Review by rchl — APPROVED** [review]

**Comment by predragnikolic** [comment]
Thanks for the review :)

**Pull request merged**